Tasks/ServiceFabricDeploy/utilities.ps1

```diff
@@ -110,6 +110,40 @@ function Read-PublishProfile
     return $publishProfile
 }
 
+function Get-OverridenApplicationParameters
+{
+    Param (
+        [String]
+        $ApplicationManifestPath
+    )
+
+    $overrideParameters = @{}
+    $applicationManifestXml = [Xml] (Get-Content -LiteralPath $ApplicationManifestPath)
+    foreach ($param in $applicationManifestXml.ApplicationManifest.Parameters.Parameter)
```

Comment:
Can there be null ref if the xml doesn't have parameters section or zero parameters

Reply:
I ran some tests for this, doesn't seem to fail. Should we continue ?

Reply:
Can you try this with a sample manifest with zero parameters and no parameters section and see this code doesn't result in failure

Reply:
I had tried and did not failed. Tried again now, the code change doesn't throw.